highfidelity/hifi
# WL 21667 - Window.browseDir() dialog doesn't let user use Esc to cancel

> Follow the behavior of Window.prompt()
> [Worklist 21667](https://worklist.net/21667)

## Test Plan
1. From the Main Menu, click File and click Import Entities
2. Confirm you can press Esc to close out of it
3. Enable Advanced/Developer Menus
4. Run `Window.browseDir()` in your console and confirm you can also Esc out of that prompt.
